test/UnitTests/StudentsDashboard.Application.UnitTests/WorkEvent/Commands/AddWorkEvent/AddWorkEventHandlerTests.cs

```diff
@@ -0,0 +1,53 @@
+using FluentAssertions;
+using Moq;
+using StudentsDashboard.Application.Persistance;
+using StudentsDashboard.Application.UnitTests.WorkEvent.TestUtils;
+using StudentsDashboard.Application.WorkEvents.Commands.AddWorkEvent;
+
+namespace StudentsDashboard.Application.UnitTests.WorkEvent.Commands.AddWorkEvent;
+
+public class AddWorkEventHandlerTests
+{
+    private readonly AddWorkEventHandler _handler;
+    private readonly Mock<IWorkEventRepository> _mockIWorkEventRepository;
+    private readonly Mock<IUserContextGetIdService> _mockIUserContextGetIdService;
+
+    public AddWorkEventHandlerTests()
+    {
+        _mockIWorkEventRepository = new Mock<IWorkEventRepository>();
```

Comment:
W ramach ciekawostki, jak wcześniej podałeś typ przy deklaracji to tutaj nie musisz ponownie pisać "new Mock< IWorkEventRepository>();", wystarczy "new()"